docs/dev-tools/zombienet/tutorials/minimal-example.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Minimal Example of the usage of Zombienet
+description: This tutorial provides an example of how to use Zombienet to spawn a basic network and run a simple test over it.
+---
+
+# A Minimal Example of the usage of Zombienet
+
+This tutorial provides an example of how to use Zombienet to spawn a basic network and run a simple test against it.
+
+## Prerequisites
+
+To follow this tutorial, first, you need to have Zombienet installed. If you haven't done so, please follow the instructions in the [Installation](../overview.md/#installation){target=_blank} section.
+
+## Defining the network
+
+As mentioned in the [Configuration Files](../overview.md/#configuration-files){target=_blank} section, Zombienet uses a configuration file to define the ephemeral network that will be spawned. To follow this tutorial, create a file named `minimal-example.toml` with the following content:
+
+```toml
```

Comment:
Marking this item as solved since it will be resolved when those folders are merged into the main branch. It will be addressed at https://opslayer.atlassian.net/browse/DRP-40